### ISSUE TITLE
Can there be direct links to heading?

### ISSUE THREAD

**Issue by sarbbottam** [code-kotis/pwa-codelabs#5]
For example, https://pwa.tips/codelabs/serviceworker#how-to-register to 

<img width="1425" alt="screen shot 2016-11-11 at 4 12 17 pm" src="https://cloud.githubusercontent.com/assets/949380/20233910/b4588bd0-a829-11e6-9dd3-5d61d514d25a.png">

Not sure if that would be possible with client-side rendering but would be nice, since this is documentation.

**Comment by gokulkrishh** [comment]
@sarbbottam Yep you are right. Since we are using react-router we cant have a `#hotlink` like above. But will see if I can find a workaround.

**Comment by sarbbottam** [comment]
I will give it a try. I implemented something similar at http://sarbbottam.github.io/empower-your-users/tab. It will nice if we can do it.
